### DISCUSSION TITLE
Multiplayer Problems

### DISCUSSION THREAD

**Discussion by achraf22-dev** [mupen64plus-ae/mupen64plus-ae#912] [Q&A]
This app has been phenomenal since I downloaded it for the first time, I bought the premium version to play with my friends but Im encountring a major problem, when we are playing mario party 3, everything is going good then suddenly the game lags and someome gets kicked out of the game to the main menu of the emulator and he cant join us to the server even tho we making sure that we have strong wifi connection. Please fix this issue as soooon as possible.

**Comment by fzurita** [comment]
It's possible that the OS may be putting the netplay thread to sleep. I'll increase the priority of the thread and hopefully that addresses it.
